external_modules/html2canvas/.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "src/fabric"]
+	path = src/fabric
+	url = https://github.com/kangax/fabric.js.git
```

Comment:
`html2canvas` seems to be referencing a submodule (`kangax/fabric`) that does not exist in this source tree. Is that intentional? Or should the `external_modules` folder be entirely deleted from this PR?